README.md

```diff
@@ -42,3 +42,20 @@ Replaced:
 #contents of my_rule.rego
 #ENDINLINE
 ```
+
+### Linting Policies
+FCV provides a policy linter.  You can invoke it as:
+
+```
+go get github.com/forseti-security/config-validator/cmd/policy-tool
```

Comment:
Note it'd likely be good for us to upload a copy to GCS somewhere potentially.

Reply:
The eventual plan is to add this check as a cloudbuild step once the linter has a Dockerfile (sometime this week).